Dockerfile

```
@@ -24,11 +29,11 @@ RUN apt-get update && apt-get upgrade -y \
   # https://security-tracker.debian.org/tracker/CVE-2016-2779
   && rm -f /usr/sbin/runuser \
   # https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-6954
-  && rm -f /usr/lib/x86_64-linux-gnu/libdb-5.3.so
+  && rm -f /usr/lib/${DOCKER_ARCH}-linux-gnu/libdb-5.3.so
```

DOCKER_ARCH is `amd64`

```suggestion
  && rm -f /usr/lib/${ARCH}-linux-gnu/libdb-5.3.so
```

---

Resolved.